client/signup/config/steps-pure.js

```diff
@@ -524,6 +524,17 @@ export function generateSteps( {
 			providesDependencies: [ 'bearer_token', 'email', 'username' ],
 			unstorableDependencies: [ 'bearer_token' ],
 		},
+
+		'passwordless-woo': {
+			stepName: 'passwordless-woo',
+			props: {
+				headerText: i18n.translate( 'Connect your store' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Connecting your store' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).